opal/mca/btl/base/btl_base_am_rdma.c

```diff
@@ -409,7 +410,13 @@ static inline int mca_btl_base_am_rdma_advance(mca_btl_base_module_t *btl,
     }
 
     if (send_descriptor) {
-        return btl->btl_send(btl, endpoint, descriptor, mca_btl_base_rdma_tag(hdr->type));
+        /* This descriptor was created with MCA_BTL_DES_SEND_ALWAYS_CALLBACK set, so safe to treat
+         * OPAL_SUCCESS and 1 return codes the same */
+        assert(0 != (descriptor->des_flags && MCA_BTL_DES_SEND_ALWAYS_CALLBACK));
+        ret = btl->btl_send(btl, endpoint, descriptor, mca_btl_base_rdma_tag(hdr->type));
+        if (ret == 1)
```

Comment:
Sorry, I missed this in the previous iteration.  Open MPI *always* uses curly braces around conditional bodies.

Reply:
Thank you! fixed in new revision